pkg/specgen/generate/container.go

```diff
@@ -169,6 +170,19 @@ func CompleteSpec(ctx context.Context, r *libpod.Runtime, s *specgen.SpecGenerat
 		}
 	}
 
+	// If caller did not specify Pids Limits load default
+	if s.ResourceLimits.Pids == nil {
+		if s.CgroupsMode != "disabled" {
```

Comment:
We also need to make sure we are not rootless + cgroups v1

Reply:
The rtc.PidsLimit() takes care of this.

Reply:
rtc.PidsLimit() is returning 4096 on my cgroups v1 machine, rootless, even though the default does not show in run --help.